edgedb/protocol/protocol.pyx

```diff
@@ -150,10 +150,9 @@ cdef class SansIOProtocol:
         self.reset_status()
         self.protocol_version = (PROTO_VER_MAJOR, 0)
 
-        self.state_type_id = NULL_CODEC_ID
+        self.state_type_id = INVALID_CODEC_ID
```

Comment:
I don't think we need to do this anymore, do we?

Reply:
Right, good catch!